tests/cargo-kani/vecdeque-cve/src/harness.rs

```diff
@@ -11,9 +11,16 @@
 #![feature(ptr_internals)]
 #![feature(rustc_allow_const_fn_unstable)]
 
+#[cfg(disable_debug_asserts)]
+macro_rules! debug_assert {
+    ( $( $x:expr ),* ) => {};
+}
```

Comment:
I'm sure there's a better way using `RUSTFLAGS` but I couldn't work out a way to disable `debug_asserts` at the `cargo kani` command line.

Reply:
Kani overrides `debug_asserts` to apply the same instrumentation used for asserts: https://github.com/model-checking/kani/blob/4d4a5a7b99121f943b6d726b2c559767209531cd/library/std/src/lib.rs#L82 so what you did is likely the only way to disable them, unless we want to support disabling debug asserts (similar to the behavior of `cargo build --release`) which we can do through defining them as follows:
```
#[macro_export]
macro_rules! debug_assert {
    ($($x:tt)*) => (if $crate::cfg!(debug_assertions) ({ $crate::assert!($($x)*); })
}
```

Reply:
Ah, I see. That's why my attempts didn't work. I tried `RUSTFLAGS='--cfg "debug_assertions=\"false\""'` and using a profile in `Cargo.toml` (https://doc.rust-lang.org/book/ch14-01-release-profiles.html):

```
[profile.dev]
opt-level = 3
```

Reply:
Would adding support for turning off `debug_assertions` help? I can push a PR to do that.